setup.cfg

```diff
@@ -13,25 +14,36 @@ classifiers =
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
     Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
     Topic :: Scientific/Engineering
 author = Wu-Jung Lee
 author_email = leewujung@gmail.com
 platforms = OS Independent
 
-[black]
-line-length = 100
+[options]
+packages = find:
+platforms = any
+py_modules =
+    _echopype_version
+include_package_data = True
+install_requires =
+  dask[array]
+  netCDF4
+  pynmea2
+  pytz
+  scipy
+  xarray
+  zarr
+  fsspec
+  s3fs
+python_requires = >=3.8
 
-[flake8]
-max-line-length = 100
-output-file = flake8_log.txt
-tee = True
-ignore = E722,E203,W503,T001
-
-[isort]
-known_first_party = echopype
-known_third_party = _echopype_version,setuptools
-multi_line_output = 3
-include_trailing_comma = True
-force_grid_wrap = 0
-combine_as_imports = True
-line_length = 100
+[options.extras_require]
+plot =
+    matplotlib
```

Comment:
In the conda  yml files for the CI, you're now using matplotlib-base. Is there a good reason for this one to be different?

Reply:
matplotlib-base is a conda package. It's only matplotlib in pypi

Reply:
Right, I keep forgetting that!